sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -522,15 +522,15 @@ case class DescribeTableCommand(
         throw new AnalysisException(
           s"DESC PARTITION is not allowed on a temporary view: ${table.identifier}")
       }
-      describeSchema(catalog.lookupRelation(table).schema, result)
+      describeSchema(catalog.lookupRelation(table).schema, result, isExtended)
```

Comment:
Thank you Right. This PR considered only simple DESC. I'll update 'DESC EXTENDED', too. It is different from DESC FORMATTED as you pointed.